notebooks/samples/tensorflow/prediction_logging/covertype_training_serving_logging_bq.ipynb

```diff
@@ -0,0 +1,1931 @@
+{
```

Comment:
<p>nit: Clean up PROJECT_ID and BUCKET to [your-project-id], ['your-bucket-name']</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/153/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Done